src/facets/FundsFacet.sol

```diff
@@ -286,7 +286,7 @@ contract FundsFacet is RoleCheck, PausableCheck, ERC1155SupplyUpgradeable, IFund
     }
 
     /// @inheritdoc IFundsFacet
-    function accrueFee() public notPaused onlyRole(LibRoles.FUNDS_OPERATOR) {
+    function accrueFee() public notPaused {
```

Comment:
Access control here is not needed. Next week we'll upgrade base and mainnet with that.